ballerina/tests/simple-params-query-test.bal

```diff
@@ -673,6 +673,110 @@ function queryJsonParam3() returns record {}|error? {
     validateJsonTable(queryMysqlClient(sqlQuery, resultType = JsonResult));
 }
 
+@test:Config {
+    groups: ["queryRow","query-simple-params"]
+}
+function queryRecord() returns sql:Error? {
+    int rowId = 1;
+    sql:ParameterizedQuery sqlQuery = `SELECT * from DataTable WHERE row_id = ${rowId}`;
+    Client dbClient = check new (host, user, password, simpleParamsDb, port);
+    record {} queryResult = check dbClient->queryRow(sqlQuery);
+    validateDataTableResult(queryResult);
```

Comment:
You need to close the `dbClient` after using it.

Reply:
Fixed in 095d9d5